src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.props

```diff
@@ -74,13 +74,4 @@ Copyright (c) .NET Foundation. All rights reserved.
     <None Remove="**\*.razor" />
   </ItemGroup>
 
-  <!-- 
```

Comment:
@rynowak here's the original change for reference: https://github.com/aspnet/AspNetCore-Tooling/commit/1c4b19ae88c4a4f53671b85f8afb0a6c32c0efbf

Components.targets gets conditionally imported for RazorLangVersion > 3.0, however the design time targets and these properties are always declared (before this change). Moving the property in Components.targets means that these properties are null when you're targeting a 2.1 project with the newer SDK. 

Scenarios I verified 

1) 2.1 project with the newer design time targets, older SDK
2) 2.1 project without design time targets
3) 3.0 project with the newest SDK